components/molecules/HighlightInput/highlight-input-form.tsx

```diff
@@ -541,12 +541,21 @@ const HighlightInputForm = ({ refreshCallback }: HighlightInputFormProps): JSX.E
           <form onSubmit={handlePostHighlight} className="flex flex-col gap-4 font-normal">
             <p role="alert">
               {errorMsg && (
-                <span className="inline-flex items-center gap-2 px-2 py-1 text-red-500 bg-red-100 border border-red-500 rounded-md w-full text-sm">
+                <span className="inline-flex items-center gap-2 px-2 py-1 mt-2 text-red-500 bg-red-100 border border-red-500 rounded-md w-full text-sm">
                   <MdError size={20} /> {errorMsg}
                 </span>
               )}
+              {!hasValidCharCount && (
+                <span className="inline-flex items-center gap-2 px-2 py-1 mt-2 text-red-500 bg-red-100 border border-red-500 rounded-md w-full text-sm">
```

Comment:
@FatumaA, the colors being used cause colour contrast issues.

![CleanShot 2024-02-05 at 07 57 14](https://github.com/open-sauced/app/assets/833231/17a391ea-bee2-4195-844c-825e5edfa245)

![CleanShot 2024-02-05 at 07 58 42](https://github.com/open-sauced/app/assets/833231/f885f0ab-74e3-4b13-ab10-237380e19570)

`bg-red-100` should be white instead.

```suggestion
                <span className="inline-flex items-center gap-2 px-2 py-1 mt-2 text-red-500 bg-white border border-red-500 rounded-md w-full text-sm">
```